O365/calendar.py

```diff
@@ -958,9 +961,13 @@ def to_api_data(self, restrict_keys=None):
         if self.__recurrence:
             data[cc('recurrence')] = self.__recurrence.to_api_data()
 
-        if self.has_attachments:
-            data[cc('attachments')] = self.__attachments.to_api_data()
-
+#         if self.has_attachments:
+#             data[cc('attachments')] = self.__attachments.to_api_data()
+#  Event object doesn't have __attachments attribute...
```

Comment:
`Event` object does have the "__attachments" object. So this should be included (not commented out)